Classes/MutVoteSys.uc

```diff
@@ -1238,10 +1236,16 @@ function VS_MapList LoadMapListByPrefix(string Prefix) {
 	MapName = FirstMap;
 
 	do {
-		ML.AddMap(Left(MapName, Len(MapName) - 4)); // we dont care about extension
+		ML.AddMap(CleanMapName(MapName));
```

Comment:
While not wrong, we know for sure that the map will have .unr at the end. CleanMapName thus does an unnecessary check which will execute more bytecode instructions.

This is not a deal breaker to me. I just thought i should outline my reason for not doing this before.

Reply:
That's fair. Usually I reuse code as much as possible with other languages, so logic is in one place, but I will revert this.